src/features/controls.ts

```diff
@@ -68,12 +68,16 @@ const PHI = 2.0 * Math.PI;
 const AZIMUTHAL_QUADRANT_LABELS = ['front', 'right', 'back', 'left'];
 const POLAR_TRIENT_LABELS = ['upper-', '', 'lower-'];
 
+const OFFSET_ROTATION_MULTIPLIER = 0.005;
+
 export const DEFAULT_INTERACTION_PROMPT_THRESHOLD = 3000;
```

Comment:
I see a small bug where the model seems to jerk when the prompt first comes up. I believe this is because the model starts to auto-rotate just before the prompt comes up, and then the prompt resets the rotation to zero, which could get worse if this threshold attribute is increased.

Reply:
What do you think the best way to handle auto-rotate is?

We could:
* Not play the auto-rotate animation if the interaction prompt is going to be displayed.
* Have the auto-rotate animation play as normal and animate the model back to the centre once the interaction prompt animation begins.
* The interaction prompt animation animates the model at whatever rotation it happens to be at.

Reply:
Perhaps an even simpler answer? How about make the auto-rotation delay (which is not currently configurable) equal to the interaction prompt threshold plus enough delay to ensure the prompt always fires first. @cdata Does this sound good to you?

Reply:
First, I would like to state plainly that I increasingly desire for us to deprecate `auto-rotate` in favor of #738 because the way it works is kind of annoying and I think everyone would be happier without it.

Imagining the user intention of someone literally writing this HTML:

```html
<model-viewer auto-rotate></model-viewer>
```

They have explicitly asked for auto-rotation, regardless of interaction-prompt behavior. I think bullet #3 from @maaslalani is what I expect in that case. Auto-rotation should happen as normal, and the wiggle position should be additive to whatever the turntable rotation value happens to be in a given frame.

Reply:
A potentially easier way to achieve bullet #3 would be to animate the camera instead of the pivot, since this way they would be independent and their effects would add automatically.

Reply:
This would be a breaking change, and also make auto-rotate even more annoying and useless (because it's just sugar over `camera-orbit` at that point). Orbiting the camera != rotating the pivot.

Reply:
I may have misunderstood @elalish . I now understand that he was referring to the interaction prompt, not `auto-rotate`.

So, yeah, I agree, that would be a reasonable path forward as well.